scripts/marketing_data_test.py

```diff
@@ -56,7 +56,8 @@ def scale_marketing_data(X: pd.DataFrame, T: pd.DataFrame, log_scale_skew_thresh
 
     # TODO: Update to use log scaling too. Why not?
     tscale = StandardScaler()
-    T = tscale.fit_transform(T)
+    T_scaled = tscale.fit_transform(T)
+    T = pd.DataFrame(T_scaled, index=T.index, columns=T.columns)
```

Comment:
`T` became an issue down the road, especially after `pd.concat()` with `X` since it's a numpy array object. So I'm converting it back to `pd.DataFrame` right from the beginning.